rswag-specs/spec/rswag/specs/swagger_formatter_spec.rb

```diff
@@ -124,19 +143,55 @@ module Specs
                   }
                 }
               }
-            )
+            }
+          }
+
+          it 'converts query and path params, type: to schema: { type: }' do
+            expect(config_swagger_docs[swagger_doc].slice(:paths)).to match(expected_paths)
+          end
+
+          context "when swagger_docs contains one of the docs" do
```

Comment:
Re above, not sure which style they use, but suggest you are consistent at least?